flink-cdc-connect/flink-cdc-source-connectors/flink-sql-connector-tidb-cdc/pom.xml

```diff
@@ -60,6 +60,12 @@ under the License.
                                 </includes>
                             </artifactSet>
                             <relocations>
+                                <relocation>
+                                    <pattern>org.apache.kafka</pattern>
+                                    <shadedPattern>
+                                        com.ververica.cdc.connectors.shaded.org.apache.kafka
```

Comment:
Maybe we should change `  com.ververica.`  to `org.apache.flink` after we donated to Apache?

Reply:
You're right. I didn't notice the package name had changed. Hold on a sec.

Reply:
I found no shaded pattern has migrated from `com.ververica.cdc.connectors.shaded` yet. Maybe we should do it all together in an other PR?

Reply:
> I found no shaded pattern has migrated from `com.ververica.cdc.connectors.shaded` yet. Maybe we should do it all together in an other PR?

See https://github.com/apache/flink-cdc/pull/3110.

Reply:
The shaded pattern is updated. Thanks.